cfme/tests/configure/test_start_page.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import fauxfactory
+
+from cfme.base.credential import Credential
+from cfme.configure.settings import Visual
+from cfme.fixtures.tag import role  #noqa
```

Comment:
**REQUIRED**
You don't need to import a fixture to use it in the module.

Reply:
Okay